site-generator/src/main/java/org/jboss/wildscribe/site/SingleVersionGenerator.java

```diff
@@ -168,6 +189,26 @@ private void createResourcePage(ModelNode model, Template template, boolean hasL
 
     }
 
+    private void appendSearchIndex(final ResourceDescription resourceDescription, final String url, final String relativeUrl) throws IOException {
```

Comment:
We should create a list of the expected types we want to be indexed. For example subsystem attributes should be different than operation attributes. We should also capture operations and resource paths too.